### PR TITLE
Resize compile

### PR DESCRIPTION
**Pull request by edgarriba**
#### Changes

verify that crop and resize ops work with compile

#### Type of change

- [ ] 📚  Documentation Update
- [x] 🧪 Tests Cases
- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

File: kornia/geometry/bbox.py [hunk 1]
```diff
@@ -37,16 +37,19 @@ def validate_bbox(boxes: torch.Tensor) -> bool:
     if len(boxes.shape) == 4:
         boxes = boxes.view(-1, 4, 2)
 
-    if not torch.allclose((boxes[:, 1, 0] - boxes[:, 0, 0] + 1), (boxes[:, 2, 0] - boxes[:, 3, 0] + 1), atol=1e-4):
+    x_tl, y_tl = boxes[..., 0, 0], boxes[..., 0, 1]
```
Comment (johnnv1) [inline kornia/geometry/bbox.py:40]: these changes get red lights on ci

File: tests/geometry/transform/test_affine.py [hunk 2]
```diff
@@ -177,6 +177,14 @@ def test_gradcheck(self, device):
         input = torch.rand(1, 2, 3, 4, device=device, dtype=torch.float64)
         self.gradcheck(kornia.geometry.transform.Resize(new_size, align_corners=False), (input,))
 
+    @pytest.mark.parametrize("anti_alias", [True, False])
+    def test_dynamo(self, device, dtype, anti_alias, torch_optimizer):
+        new_size = (5, 6)
+        input = torch.rand(1, 2, 3, 4, device=device, dtype=dtype)
+        op = torch_optimizer(kornia.geometry.transform.resize)
+        out = op(input, new_size, align_corners=False, antialias=anti_alias)
+        assert out.shape == (1, 2, 5, 6)
```
Comment (johnnv1) [inline tests/geometry/transform/test_affine.py:186]:
```suggestion
        assert out.shape == (1, 2, 5, 6)
        expected = kornia.geometry.transform.resize(input, new_size, align_corners=False, antialias=anti_alias)
        self.asser_close(out, expected)
```

File: kornia/geometry/bbox.py (hunk 1 above)
Comment (edgarriba) [reply kornia/geometry/bbox.py:40]: should be fixed now

File: tests/geometry/transform/test_affine.py (hunk 2 above)
Comment (edgarriba) [reply tests/geometry/transform/test_affine.py:186]: done

**Review by johnnv1 — APPROVED** [review]

**Pull request merged**